packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -268,17 +291,17 @@ class AOTSnapshotter {
     const String embedBitcodeArg = '-fembed-bitcode';
     final String assemblyO = _fileSystem.path.join(outputPath, 'snapshot_assembly.o');
 
-    final RunResult compileResult = await _xcode.cc(<String>[
+    RunResult result = await _xcode.cc(<String>[
```

Comment:
I think the multiple locals with different names was clearer.

Reply:
✔️